src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -17,6 +17,8 @@ export default class MultipleProjectSelector extends React.Component {
     projects: PropTypes.array,
     onChange: PropTypes.func,
     onUpdate: PropTypes.func,
+    align: PropTypes.oneOf(['right', 'left']),
```

Comment:
Do we still need to support the right aligned project selector anymore? I thought it was just for Discover